pack/cc.json

```diff
@@ -111,9 +111,9 @@
         "quantity": 3,
         "side_code": "runner",
         "strength": 0,
-        "stripped_text": "If you have at least 2 link, the memory cost of Crowbar is 0, even if it is not installed. Crowbar has +1 strength for each installed icebreaker. trash: Break up to 3 code gate subroutines.",
+        "stripped_text": "If you have at least 2{link}, the memory cost of this program is 0 mu, even if it is not installed. This program gets +1 strength for each installed icebreaker. Interface -> trash: Break up to 3 code gate subroutines.",
```

Comment:
`stripped_text` shouldn't contain any braces/brackets

Reply:
Oops, missed link when processing the brackets.